### PR TITLE
Add trackBulk for track bulk events

### PR DESCRIPTION
**Pull request by yantze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,24 @@ matomo.track({
...
+  'e_a': 'rightButton',
+  'e_v': '2'
+},{
+  '_id': 'AA814767-7B1F-5C81-8F1D-8E47AD7D2982',
+  'cdt': '2018-03-22T02:33:52.962Z',
+  'e_c': 'Buy',
+  'e_a': 'leftButton',
+  'e_v': '4'
+}];
+matomo.track(events, (resData) => {
```
Comment (Findus23) [inline README.md]: I think this should read `matomo.trackBulk`? Or am I missing something?

**Comment by Findus23** [comment]
Hi, this is looking great. Sorry that I didn't have time to look into it sooner.

Minor nitpick: I think the function in the README should read `matomo.trackBulk`.

File: README.md (hunk 1 above)
Comment (yantze) [reply README.md]: Sorry, i will update soon. :)

**Comment by yantze** [comment]
Hi, I have updated the `README.md` file.

**Comment by Findus23** [comment]
Many thanks for your contribution.
https://github.com/matomo-org/matomo-nodejs-tracker/pull/47/commits/73b5df0a3964465aa748a4215266674ef9046d9e reminded me to fail the test if eslint shows warnings
(https://github.com/matomo-org/matomo-nodejs-tracker/commit/f2b4c2a172e75045a6f08de3eed1e5cbcb7a6614)

**Comment by Findus23** [comment]
I have published 2.2.0 with your changes.

**Pull request merged**